new_dawn_server/users/api/tests/test_user_answer_question_registration.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from new_dawn_server.questions.models import AnswerQuestion, Question
+from new_dawn_server.users.models import Profile
+from tastypie.test import ResourceTestCaseMixin
+
+
+class AnswerQuestionInRegisterTest(ResourceTestCaseMixin, TestCase):
+    def setUp(self):
+        super().setUp()
+        self.register_arguments = {
+            "first_name": "test",
+            "last_name": "user",
+            "username": "test-user",
+            "password": "test-pwd",
+        }
+        self.account_arguments = {
+            "birthday": "1990-01-01",
+            "phone_number": "+12345678900",
+            "gender": "M",
+        }
+        self.profile_arguments = {
+            "description": "nice",
+            "employer": "MANMAN",
+            "hometown": "NY",
+            "job_title": "CEO",
+            "profile_photo_url": "www",
+            "school": "NYU",
+            "smoke": True,
+        }
+        self.answer_question_arguments = {"answer_question":
```

Comment:
Let's also create a Question object (assuming the question object is created by admin) and test if you can get the same object when storing answer question.